components/general/Hero.tsx

```diff
@@ -13,7 +13,10 @@ const Hero = () => {
             digital agency
             <br /> from Melbourne.
           </div>
-          <div className={`${textFont.className} global__text-xlarge ${styles["hero__txt"]}`}>
+          <div
+            className={`${textFont.className} global__text-xlarge ${styles["hero__txt"]}`}
+            aria-label="hero__txt"
```

Comment:
You use this aria-label in the wrong way. 
1. I don't see any reason to use aria-label on this div.
2. aria-label should be a description or the meaning of this element, computer or mobile phone can not read text "hero__txt".

Reply:
Yes, it's not a very proper way. Any suggestions to select this element except getByText since this text is repeating all over the place. Is getAllByText[index] and getByTitle better? I'm not sure about this.

Reply:
You may have missed the Testing playground tutorial.

You can find the recorded video on my Youtube channel or try this tool to see how it goes.
https://testing-playground.com/